src/test/java/HttpUrlConnectionTest.java

```diff
@@ -25,7 +27,7 @@
 
 import static com.easypost.easyvcr.internalutilities.Tools.readFromInputStream;
 
-public class HttpUrlConnectionTest {
+public class  HttpUrlConnectionTest {
```

Comment:
```suggestion
public class HttpUrlConnectionTest {
```
extra space

Reply:
Not worth nuking the approvals over it. We'll clean it up in the future.